### ISSUE TITLE
NVDA does not read notifications in Windows 10

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6013]
Notifications which I refer are those when a hardware such as USB stick are connected or may already be eject These notifications pegylated read previously.
windows 10.0.10586, NVDA latest version next.

**Comment by josephsl** [comment]
Hi, Do you remember when was the last time NVDA read them? I’ll keep a close eye on this.

**Comment by fernando-jose-silva** [comment]
Thank you for return, I do not remember precisely which version that worked correctly, most believe that the next version 13320 this problem does not happen so often.

 

De: josephsl [mailto:notifications@github.com] 
Enviada em: sábado, 28 de maio de 2016 13:12
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] NVDA does not read notifications in Windows 10 (#6013)

 

Hi, Do you remember when was the last time NVDA read them? I’ll keep a close eye on this.

**Comment by ehollig** [comment]
CC @josephsl and @fernando-jose-silva, is this still the case? I found that Windows 10 did not announce the notification when I had the announcement of tooltips turned off but it works fine when that option is checked.

**Comment by fernando-jose-silva** [comment]
Thank you very much, I do not have this problem any more.

**Comment by ehollig** [comment]
Closing from https://github.com/nvaccess/nvda/issues/6013#issuecomment-322896956
